src/utils.js

```diff
@@ -0,0 +1,44 @@
+module.exports = {
+  /**
+   * Just like parseInt, but fixed to base-10
+   */
+  Int: function (str) {
+     return parseInt(str, 10);
+  },
+
+  /**
+   * Cycles through an object and changes all numeric fields to ints
+   * where necessary. 'context' is used for exception reporting and can be
+   * left unset upon invocation.
+   */
+  cleanJSON: function cleanJSON(json, context) {
+     if (!context)
+        context = "root";
+
+     var integerProperties = [
+        'x'
+       ,'y'
+       ,'width'
+       ,'height'
+       ,'radius'
+       ,'strokeWidth'
+     ];
+     const isIntegerProperty = (property) => integerProperties.indexOf(property) != -1;
+
+     for (var property in json) {
+        if (typeof(json[property]) == 'object') {
+           cleanJSON(json[property], context + '.' + property);
+        }
+        else if (isIntegerProperty(property)) {
+           if (typeof(json[property]) == 'string') {
+            json[property] = parseInt(json[property]);
```

Comment:
I think one of the copies was using `Int` here, and the other had `parseInt` with no base specified (which means it extracts it from the string). We should probably use the equivalent of `Int`:
```suggestion
            json[property] = parseInt(json[property], 10);
```

Reply:
Oof. Hopefully that duplication isn't semantic..